sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryStatusAndProgressSuite.scala

```diff
@@ -215,6 +218,60 @@ class StreamingQueryStatusAndProgressSuite extends StreamTest with Eventually {
     }
   }
 
+  test("SPARK-29973: Make `processedRowsPerSecond` calculated more accurately and meaningfully") {
```

Comment:
The CI build fails here, and in fact this UT doesn't test the actual change. Wouldn't it pass the test with current master branch?

Actually it can't be tested unless modifying clock to support sub-milliseconds (say, microseconds), but it would bring too many change compared to the actual change we do.  (Still be able to test with 0 milliseconds, but that's slightly different than what we intended.)

Personally I'm OK to skip adding the UT in this case, but may need to hear others' voices as well.

Reply:
- The latest commit is `50baee9` and it fails PySpark unit tests. I think it is irrelevant. 
- In this test, the batch will end with same time with batch begin. And it test the  actual change.
- The change in PR is explicit, and there may not be a need to add UT.

Reply:
Thanks for explaining. I realized that the build failure was from old commit; and now I'm seeing the intention of the test - 0 milliseconds batch, which seems OK. Please follow the other comments - there's copy & paste which I would like to see it be reused.